driver-examples/osgi/README.rst

```diff
@@ -0,0 +1,32 @@
+OSGi Example
```

Comment:
Created this as a separate project as I didn't want to polute driver-core with test scoped dependencies, hope that is ok.